lib/attribute.js

```diff
@@ -455,7 +456,10 @@ validators.format = function validateFormat (instance, schema, options, ctx) {
   if (typeof instance === 'undefined') {
```

Comment:
changing if (instance === undefined || instance === null) would change the behavior for instance == null from returning false to returning null. I was not sure if this would be a breaking change. If you think it should be fine, then I will make the change